akka-typed/src/main/scala/akka/typed/MessageAndSignals.scala

```diff
@@ -63,4 +63,8 @@ final case class Terminated(ref: ActorRef[Nothing])(failed: Throwable) extends S
   def wasFailed: Boolean = failed ne null
   def failure: Throwable = failed
   def failureOption: Option[Throwable] = Option(failed)
+
+  /** Java API */
+  def getRef(): ActorRef[Nothing] = ref
```

Comment:
Hmm yeah `Nothing` is a scala-ism ofc... though wouldn't `ActorRef[Void]` suggest you can send it a `null`? Probably fine though, we have lots of API's that strictly speaking accept `null` but will definitely error ;).